src/ocean/io/select/client/TimerSet.d

```diff
@@ -256,15 +256,7 @@ public class TimerSet ( EventData ) : TimerEventTimeoutManager
 
     ***************************************************************************/
 
-    private ObjectPool!(Event) events;
-
-    version (UnitTest)
-    {
-        public size_t allocated_event_count ( )
-        {
-            return events.length();
-        }
-    }
+    protected ObjectPool!(Event) events;
```

Comment:
Changing this field from private to protected seems unrelated to "Remove unnecessary unittest function".

Reply:
Good catch, moved to the next commit